AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -69,7 +68,13 @@ public final class UPIComponent: PaymentComponent,
     
     internal private(set) var currentSelectedItemIdentifier: String?
 
-    internal private(set) var selectedUPIFlow: UPIFlowType = .upiApps
+    /// Represents the selected UPI (Unified Payments Interface) flow for the payment component.
+    /// Determines the specific UPI transaction process to follow.
+    ///
+    /// Possible values are defined by `UPIFlowType`:
```

Comment:
This comment doesn't seem to be necessary - the info should be available on the case documentation of the `UPIFlowType`

Reply:
Done